Assets/Scripts/UI/Menu's/PauseController.cs

```diff
@@ -0,0 +1,52 @@
+using UnityEngine;
+
+public sealed class PauseController : MenuController
+{
+    [SerializeField] private GameObject activePauseMenu;
+    [SerializeField] private GameObject activePauseScreen;
+    [SerializeField] private GameObject activeOptionsScreen;
+
+    private bool _canPause = true;
+    private bool _isNotPaused = true;
```

Comment:
```suggestion
    private bool _isPaused;
```
Ik zou het woord "Not" weg halen, ziet er wat netter en duidelijker uit. Je moet dan welk alle logica van deze bool moeten opdraaien (waar het true is moet false zijn en anders om).